module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/guest/repository/GuestRepository.kt

```diff
@@ -34,4 +35,14 @@ class GuestRepository(
 
     override fun getReceivedInvitations(member: Member) =
         guestJPARepository.getReceivedInvitations(MemberEntity.from(member)).map(InvitationEntity::toDomain)
+
+    override fun findAttendGuestsByInvitation(invitation: Invitation): List<Guest> {
+        return guestJPARepository.findAttendGuestsByInvitation(InvitationEntity.from(invitation))
+            .map { it.toDomain() }
+    }
+
+    override fun findNotAttendGuestsByInvitation(invitation: Invitation): List<Guest> {
+        return guestJPARepository.findNotAttendGuestsByInvitation(InvitationEntity.from(invitation))
+            .map { it.toDomain() }
```

Comment:
마찬가지로, 메서드 참조가 더욱 간결할 것 같아보이네요.
```suggestion
            .map(GuestEntity::toDomain)
```

Reply:
마찬가지로 수정하였습니다!!

Reply:
마찬가지로 수정하였습니다!!